demos/common/samples/common.hpp

```diff
@@ -1114,11 +1114,14 @@ inline std::size_t getTensorBatch(const InferenceEngine::TensorDesc& desc) {
 inline void showAvailableDevices() {
     InferenceEngine::Core ie;
     std::vector<std::string> devices = ie.GetAvailableDevices();
-
-    std::cout << std::endl;
-    std::cout << "Available target devices:";
-    for (const auto& device : devices) {
-        std::cout << "  " << device;
+    if (!devices.empty()) {
+        std::cout << std::endl;
+        std::cout << "Available target devices:";
+        for (const auto& device : devices) {
+            std::cout << "  " << device;
+        }
+    }
+    else {
+        THROW_IE_EXCEPTION << "Not available any target device to infer on";
```

Comment:
How could this be possible? I'd assume that at least CPU is always available.

Reply:
Yeah, it supposed to be, I just revert a walkaround commit of Ilya.

Reply:
As I remember, @ilya-lavrenov said there is a config in IE that allows to disable plugins.

Reply:
Alright.